x-pack/plugins/observability_ai_assistant/public/hooks/use_timeline.test.ts

```diff
@@ -197,10 +198,16 @@ describe('useTimeline', () => {
       });
 
       it('adds two items of which the last one is loading', async () => {
```

Comment:
Can you also add a test here for the collapsing behaviour? (I assume this is the right place to do that, but not sure).